pep-0676.rst

```diff
@@ -74,38 +77,42 @@ There are several requests for additional features in reading PEPs, such as:
 * support for SVG images [3]_
 * typographic quotation marks [4]_
 * additional footer information [5]_
-* intersphinx functionality [6]_
+* Intersphinx functionality [6]_
 
 These are "easy wins" from this proposal, and would serve to improve the
 quality-of-life for consumers of PEPs (including reviewers and writers). For
-example, the reference implementation no longer requires a scheduled render
+example, the new reference implementation no longer requires a scheduled render
 process to run, instead initiating rendering on every commit to the
 `python/peps`_ repository.
 
-Equally, there are a small number of broken items, for example list styles not
-being respected or support for updating images being challenging with the
-system [7]_. These would be solved by default in the proposal.
+Equally, there are a number of broken items currently; for example, list styles
+are not respected, and support for updating images is challenging with the
+current system [7]_. These are solved by the reference implementation of
+this proposal.
 
-Commercial providers such as `Read the Docs`_ can additionally enhance this
-experience, for example by providing rendered previews of changes in pull
+Third-party providers such as `Read the Docs`_ can additionally enhance this
+experience; for example, by providing rendered previews of changes in pull
 requests.
 
+
 Specification
 =============
 
```

Comment:
They are widely used in the specification section of other PEPs, and I've never seen it explicitly noted like this. Perhaps it would be a better idea, if this is desired, to do so in PEP 1 or PEP 12 instead, instead of redundantly including this note in every PEP.